stdlib/LinearAlgebra/src/matmul.jl

```diff
@@ -217,18 +217,24 @@ end
 """
     mul!(Y, A, B) -> Y
 
-Calculates the matrix-matrix or matrix-vector product ``AB`` and stores the result in `Y`,
+Calculates the matrix-matrix or matrix-vector product ``A B`` and stores the result in `Y`,
```

Comment:
The whitespace here is to make the style consistent with that in the 5-term docstring